src/shared/components/case-viewer/case-full-access-view/case-full-access-view.component.ts

```diff
@@ -118,7 +118,17 @@ export class CaseFullAccessViewComponent implements OnInit, OnDestroy, OnChanges
         this.activitySubscription = this.postViewActivity().subscribe();
       });
     }
-
```

Comment:
My only comment is that I think this should be split out into a method, there is already a lot going on in this ngOnit().. just create a method - checkRouteAndSetCaseViewTab() and call it from in here.